Do not require cryptography. It brings in an unnecessary dependency on rust.
Reverts part of https://github.com/mongodb/mongo/commit/eb11991ff4fd8340688b66878db561ee61943a2c
--- etc/pip/components/platform.req.orig	2023-01-11 13:46:46.000000000 -0600
+++ etc/pip/components/platform.req	2023-01-28 20:14:11.000000000 -0600
@@ -1,6 +1,3 @@
 # Platform-specific components
 pypiwin32>=223; sys_platform == "win32" and python_version > "3"
 pywin32>=225; sys_platform == "win32" and python_version > "3"
-
-cryptography == 2.3; platform_machine == "s390x" or platform_machine == "ppc64le" # Needed for oauthlib to use RSAAlgorithm # Version locked - see SERVER-36618
-cryptography == 36.0.2; platform_machine != "s390x" and platform_machine != "ppc64le"
\ No newline at end of file
